partner_contact_age/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+Adds a computed age field on Partners
+
+.. image:: /partner_contact_age/static/description/age.png
```

Comment:
```suggestion
.. image:: ../static/description/age.png
```

Use this relative path style for pictures, so that it works well everywhere: apps store, odoo, etc..